src/styles/theme/oc-button.scss

```diff
@@ -28,7 +28,7 @@
 
   text-transform: none !important;
 
-  &:not(.uk-button-small):not(.uk-button-large) {
+  &:not(.uk-button-small):not(.uk-button-large):not(.uk-button-raw) {
```

Comment:
Since it is possible to set a size value and a variation separately, you should add `:not(.uk-button-raw)` to the other two classes (`.uk-button-small` and `.uk-button-large`) as well. I know that it's stupid, but just to be sure.

Reply:
done. please check my alternative solution

Reply:
Nice, even better!